pkg/hoist/test_helper.go

```diff
@@ -29,11 +30,24 @@ func FakeHoistLaunchableForDir(dirName string) *Launchable {
 		launchable.RunAs = curUser.Username
 	}
 
-	return launchable
+	sbTemp, _ := ioutil.TempDir("", "fakesvdir")
```

Comment:
This test helper assumes that the inability to make tempdirs will be reflected by other failures